### PR TITLE
Adds N/A in unit note panel to CAUC with no expiry date

### PR DESCRIPTION
**Pull request by orelbn**
*Issue #:* /bcgov/entity#16833

*Description of changes:*
- Adds N/A for CAUC with no expiry date
- Adds tests

image

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1493-0qkwvyxo.web.app

File: ppr-ui/src/components/unitNotes/UnitNoteContentInfo.vue [hunk 1]
```diff
@@ -12,14 +12,17 @@
       </v-col>
     </v-row>
 
-    <v-row v-if="note.expiryDateTime" no-gutters class="my-7">
+    <v-row v-if="isNoticeOfCautionOrRelatedDocType(note)" no-gutters class="my-7">
```
Comment (cameron-eyds) [inline ppr-ui/src/components/unitNotes/UnitNoteContentInfo.vue:15]: This function name is a bit too short, could we make it longer please?

**Review by cameron-eyds — APPROVED** [review]
Looks fantastic

**Review by doug-lovett — APPROVED** [review]
LTGM.

File: ppr-ui/src/components/unitNotes/UnitNoteContentInfo.vue (hunk 1 above)
Comment (orelbn) [reply ppr-ui/src/components/unitNotes/UnitNoteContentInfo.vue:15]: Would `isNoticeOfCautionOrRelatedDocumentTypeLikeContinuedNoticeOfCautionOrExtendedNoticeOfCautionDocumentTypes`

work?

Comment (cameron-eyds) [reply ppr-ui/src/components/unitNotes/UnitNoteContentInfo.vue:15]: Seems much more appropriate YES!

**Pull request merged**